server/routes/admin/admin.js

```diff
@@ -5,7 +5,10 @@ module.exports = () => (router) => {
   router.use(authorisationMiddleware)
 
   router.get('/', (req, res) => {
-    return res.render('admin/index', { exitUrl: links.exitUrl })
+    return res.render('admin/index', {
+      exitUrl: links.exitUrl,
+      showExitUrl: req.user && req.user.isPrisonUser,
```

Comment:
meh.. would assume you can do `req.user?.isPrisonUser` (but have never tried it)